ansible/roles/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Is this meant to be a role? I've never seen anyone call a role `tasks`. If I were trying to pick this up with no knowledge other than Ansible knowledge this could be pretty confusing for a minute.